Add note about existing UUID for saving. (#6736)
cakephp/docs: en/orm/saving-data.rst

* ``defaults`` Set to ``false`` to not set ``$search`` properties into the
  created entity.

> Creating with existing primary key
> ==================================

Reviewer: ```suggestion
Creating with an existing primary key
=====================================
```

> Especially around non-integer primary keys (aka ``UUID``) one sometimes needs to provide an existing
> key and does not want one to be created for them.

Reviewer: ```suggestion
When handling UUID primary keys you often want to provide an externally generated value, and not have
an an identifier generated for you.
```

I think this communicates the same idea in a more direct way.

> In this case make sure you are not passing the ID as part of the marshalled data, as it should never
> be part of that. Instead, manually assign the primary key here and then patch in the rest::

Reviewer: ```suggestion
In this case make sure you are not passing the primary key as part of the marshalled data.
Instead, assign the primary key and then patch in the remaining entity data::
```
